generate-fit-profile/src/main.rs

```diff
@@ -76,6 +78,12 @@ fn run() -> Result<(), Box<dyn std::error::Error>> {
     write_decode_file(&profile, &mut out_file)?;
     rustfmt(&decode_fname);
 
+    let messages_fname = dest_dir.join("messages.rs");
+    eprintln!("Generating file: {:?}", &messages_fname);
+    let mut out_file = File::create(&messages_fname)?;
+    write_messages_file(&profile, &mut out_file)?;
+    rustfmt(&messages_fname);
```

Comment:
What are your thoughts on using [prettyplease](https://github.com/dtolnay/prettyplease) instead of rustfmt? Based on the description in the README it sounds like it might be perferable but I don't have any experience with it.

Reply:
I also saw it mentioned in the `quote` documentation, but I haven’t used it before either.  I think it’s most beneficial if is used in a build script that runs for every build.  In our case, the code generation happens offline, so just calling `rustfmt` is probably fine.  And it allows us to just run `cargo fmt` on the entire project without changing the generated code.  So I would probably keep `rustfmt`, but I don’t have a strong opinion on it.

Reply:
That's true, rustfmt it is.